modules/installation-vsphere-multiple-nics.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-vsphere-installer-provisioned-network-customizations.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-vsphere-multiple-nics_{context}"]
+= Configuring multiple NICs
+
+For scenarios requiring multiple NICs, {product-title} supports configuration of multiple network adapters per node. 
+
+:FeatureName: Configuring multiple NICs
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+
+. Specify the network adapter names in the networks section of `platform.vsphere.failureDomains[*].topology` as shown in the following file:
++
+[source,yaml]
+----
+platform:
+  vsphere:
+    vcenters:
+      ...
+    failureDomains:
+    - name: <failure_domain_name>
+      region: <region_tag_1>
+      zone: <zone_tag_1>
+      server: <fully_qualified_domain_name>
+      topology:
+        datacenter: <data_center_1>
+        computeCluster: "/<data_center_1>/host/<cluster1>"
+        networks: # <1>
+        - <VM_network1_name>
+        - <VM_network2_name>
+        - ...
+        - <VM_network10_name>
+----
+<1> Specifies the list of network adapters. You can specify up to 10 network adapters.
+
+. Define either the `networking.machineNetwork` configuration, or the `nodeNetworking` configuration after creating the manifest files.
++
+[NOTE]
+====
+The `networking.machineNetwork.cidr` must align with an address on the first adapter defined in `topology.networks`.
+====
+
+1. Do we need `_1` in the fields? Can we match the fields specified in sample install-config.yaml?
+2. nodenetworking comes as part of phase 2 i.e. after manifest files are created, whereas machineNetwork is in phase 1 i.e before manifests are created. how to handle this in doc?
```

Comment:
those fields are there but we never documented them.  we didnt have strong use case for them at the time but we do now